App/appReducer.js

```diff
@@ -12,6 +14,7 @@ const appReducer = combineReducers({
   [POSITIONS_NAMESPACE]: positionsReducer,
   [UI_NAMESPACE]: uiReducer,
   [FAVORITES_NAMESPACE]: favoritesReducer,
+  [SURVEY_NAMESPACE]: surveyReducer,
```

Comment:
The match reducer already holds survey questions and user responses.  Can you add the functionality of updating/adding a response to this reducer rather than creating anew one?